Add Japanese translation

miekof opened the pull request:
* Add Japanese translation pages
* Add translation notes for each pages

miekof committed:
Add Japanese translation

* Add Japanese translation pages
* Add translation notes for each pages

diemol commented on docs_source_files/config.toml:
```diff
@@ -167,3 +167,30 @@ name = "<i class='fab fa-fw fa-github'></i> GitHub repo"
+
+[Languages.ja]
+title = "セレニウムドキュメント"
+weight = 1
```
Could you please change the weight to `6`? So it follows the current order.

miekof replied:
Updated the weight value based on the review. 
I also noticed when referencing Selenium as a browser automation tool, "Selenium" is used instead of セレニウム. I change the title to reflect this.

diemol reviewed (changes requested):
Thank you for adding the file structure @miekof!
Just a small comment and then we can merge this.

miekof committed:
Update weight value and Selenium

* Update weight value based on the review. 
* Change セレニウム (Japanese literal expression of Selenium) to Selenium 
because it's commonly written in English when referencing to a browser 
automation tool, Selenium.

diemol reviewed (approved):
Thank you @miekof!
